packages/scene-composer/src/components/panels/scene-settings/MatterportIntegration.tsx

```diff
@@ -0,0 +1,220 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react';
+import { Box, Button, FormField, Input, Select, SpaceBetween } from '@awsui/components-react';
+import { useIntl } from 'react-intl';
+
+import useLifecycleLogging from '../../../logger/react-logger/hooks/useLifecycleLogging';
+import { useStore } from '../../../store';
+import { sceneComposerIdContext } from '../../../common/sceneComposerIdContext';
+import { KnownSceneProperty } from '../../../interfaces';
+import { getGlobalSettings, subscribe, unsubscribe } from '../../../common/GlobalSettings';
+
+export const MatterportIntegration: React.FC = () => {
+  const log = useLifecycleLogging('MatterportIntegration');
+  log?.verbose('Initialize Matterport Integration');
+
+  const intl = useIntl();
+  const sceneComposerId = useContext(sceneComposerIdContext);
+
+  const getSceneProperty = useStore(sceneComposerId)((state) => state.getSceneProperty);
+  const setSceneProperty = useStore(sceneComposerId)((state) => state.setSceneProperty);
+
+  const [dirty, setDirty] = useState(false);
+  const [matterportModelId, setMatterportModelId] = useState(getSceneProperty(KnownSceneProperty.MatterportModelId));
+  const [connectionOptions, setConnectionOptions] = useState<{ label: string; value: string }[]>([
+    {
+      label: intl.formatMessage({
+        defaultMessage: 'Select a connection',
+        description: 'Select a connection placeholder',
+      }),
+      value: 'n/a',
+    },
+  ]);
+  const [selectedConnectionName, setSelectedConnectionName] = useState('n/a');
+
+  const updateSelectedConnectionName = async () => {
+    const getSceneInfoFunction = getGlobalSettings().getSceneInfoFunction;
+    if (getSceneInfoFunction) {
+      const getSceneresponse = await getSceneInfoFunction();
+      if (getSceneresponse && getSceneresponse.sceneMetadata && getSceneresponse.sceneMetadata.MATTERPORT_SECRET_ARN) {
+        setSelectedConnectionName(getSceneresponse.sceneMetadata.MATTERPORT_SECRET_ARN);
+      } else {
+        setSelectedConnectionName('n/a');
+      }
+    }
+  };
+
+  const getConnectionList = async () => {
+    const connectionList: { label: string; value: string }[] = [
+      {
+        label: intl.formatMessage({
+          defaultMessage: 'Select a connection',
+          description: 'Select a connection placeholder',
+        }),
+        value: 'n/a',
+      },
+    ];
+    const get3pConnectionListFunction = getGlobalSettings().get3pConnectionListFunction;
+    if (get3pConnectionListFunction) {
+      const response = await get3pConnectionListFunction('AWSIoTTwinMaker_Matterport');
```

Comment:
constant for `AWSIoTTwinMaker_Matterport`

Reply:
Done - Moved to constants
`export const SECRET_MANAGER_MATTERPORT_TAG = 'AWSIoTTwinMaker_Matterport;`